src/Chain/InputProcessor/SystemPromptInputProcessor.php

```diff
@@ -28,6 +35,26 @@ public function processInput(Input $input): void
             return;
         }
 
-        $input->messages = $messages->prepend(Message::forSystem($this->systemPrompt));
+        $message = $this->systemPrompt;
+
+        if ($this->includeToolDefinitions) {
+            $tools = implode(PHP_EOL.PHP_EOL, array_map(
+                fn (Metadata $tool) => <<<TOOL
+## {$tool->name}
+{$tool->description}
+TOOL,
```

Comment:
please fix the indentation, see https://wiki.php.net/rfc/flexible_heredoc_nowdoc_syntaxes

Reply:
See 
* #236